kafka-rest/src/main/java/io/confluent/kafkarest/resources/v3/ProduceAction.java

```diff
@@ -131,15 +127,16 @@ public void produce(
 
   private CompletableFuture<ProduceResponse> produce(
       String clusterId, String topicName, ProduceRequest request, ProduceController controller) {
+    Instant requestStart = Instant.now();
 
     try {
       produceRateLimiters.rateLimit(clusterId, request.getOriginalSize());
     } catch (RateLimitExceededException e) {
+      recordRateLimitedMetrics(Duration.between(requestStart, Instant.now()));
```

Comment:
> Do we want to include rate-limited requests in the latency metric?

I don't think so. Especially in the case here where we are effectively measuring solely the rate-limiting duration.

Reply:
DOne.